shared/src/domain-logic/businessPersonaHelpers.test.ts

```diff
@@ -36,7 +36,7 @@ const generateForeignBusinessWithForeignBusinessTypeIds = (
 describe("businessPersonaHelpers", () => {
   describe("isOwningBusiness", () => {
     it("returns false when business is undefined", () => {
-      expect(isOwningBusiness(undefined)).toBe(false);
+      expect(isOwningBusiness()).toBe(false);
```

Comment:
I didn't think about passing 0 arguments when I wrote these tests... It's cleaner. 🥇